src/libraries/System.IO.FileSystem.Watcher/ref/System.IO.FileSystem.Watcher.cs

```diff
@@ -72,18 +103,25 @@ public enum NotifyFilters
     }
     public partial class RenamedEventArgs : System.IO.FileSystemEventArgs
     {
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public RenamedEventArgs(System.IO.WatcherChangeTypes changeType, string directory, string? name, string? oldName) : base (default(System.IO.WatcherChangeTypes), default(string), default(string)) { }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public string OldFullPath { get { throw null; } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public string? OldName { get { throw null; } }
     }
     public delegate void RenamedEventHandler(object sender, System.IO.RenamedEventArgs e);
     public partial struct WaitForChangedResult
     {
         private object _dummy;
         private int _dummyPrimitive;
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
```

Comment:
Why is it not on class?

Reply:
Right, it should be. I now realize the whole assembly can be skipped https://github.com/dotnet/runtime/pull/41682#issuecomment-687255619